mongodb/src/main/java/com/example/mongodb/service/UserService.java

```diff
@@ -0,0 +1,39 @@
+package com.example.mongodb.service;
+
+import com.example.mongodb.entity.User;
+import com.example.mongodb.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
```

Comment:
Instead of direactly creating a service class create a interface which has below CRUD ops, and use this class to implement the interface class